### ISSUE TITLE
[REQ] Make distinxion between movie vs anime and show vs anime show

### ISSUE THREAD

**Issue by Savanta** [giuseppe99barchetta/SuggestArr#47]
**Is your feature request related to a problem? Please describe.**
Make distinxion between movie vs anime and show vs anime show

**Describe the solution you'd like**
Tag based on anime vs movie/show

**Comment by giuseppe99barchetta** [comment]
Hi! Anime content is accessible as TV shows through the TMDB API, which allows you to retrieve anime titles seamlessly alongside regular TV shows. Therefore, it’s not necessary (and I believe it’s not currently possible) to make a distinction between the two.

If you could explain further the practical use case for your request, I would be happy to evaluate it more closely.

**Comment by PaeyMoopy** [comment]
I believe I can help as I had the same request.
Many people have their libraries separated, including myself. 

- Anime TV
- Anime Movies
- TV
- Movies

So in this case, it would be nice to separate them, as the watch history is currently pulled from everywhere, so if someone watches an Anime Movie, by default the suggestarr will pull related movies and download them to the "Movies" Library. This would then require manual intervention to move back into the proper library.

I can also see this creating duplicates, where a movie ends up in two different libraries at the same time.

**Comment by giuseppe99barchetta** [comment]
> I believe I can help as I had the same request.
> 
> Many people have their libraries separated, including myself. 
> 
> 
> 
> - Anime TV
> 
> - Anime Movies
> 
> - TV
> 
> - Movies
> 
> 
> 
> So in this case, it would be nice to separate them, as the watch history is currently pulled from everywhere, so if someone watches an Anime Movie, by default the suggestarr will pull related movies and download them to the "Movies" Library. This would then require manual intervention to move back into the proper library.
> 
> 
> 
> I can also see this creating duplicates, where a movie ends up in two different libraries at the same time.

So implementing the selection for Overseer download profile should do the trick right? I already have this in the feature request queue

**Comment by PaeyMoopy** [comment]
> So implementing the selection for Overseer download profile should do the trick right? I already have this in the feature request queue

Right, but requestarr would still need to differentiate WHERE the recommendation comes from. 
So long as it knows what library the watch history it's using came from, that should work. 

Since it's intended to be automated, it needs to know what it's recommending is "Anime" or, at the very least what it's recommending FROM is from the Anime libraries so that it can automatically assign the "AnimeTV / AnimeMovies" Overseerr profile to it.

**Comment by Savanta** [comment]
That's exactly my problem, after 36 recommendations I had to reassign folders, tags and libraries. I love the idea of this project and don't really mind doing that but streamlining the process would be amazing
